### ISSUE TITLE
Fix accessibility issues

### ISSUE THREAD

**Issue by AlainVagner** [AlainVagner/LOD4Mac#4]
This issue is related to #2. The focus here is mainly on the html of the definitions.

- [x] subtitles should be implemented as titles (`<hx>` or `role="heading"` and `aria-level="x"`)
- [x] lists should be implemented as lists (`<ul>`, `<ol>`, or `role="list"`)
